pfsimulator/parflow_lib/pf_omploops.h

```diff
@@ -25,7 +25,7 @@
  * @brief OpenMP functions and loop definitions
  */
 
-#if PARFLOW_ACC_BACKEND != PARFLOW_BACKEND_OMP
+#ifdef PARFLOW_HAVE_OMP
```

Comment:
I'm confused by this logic change.   Looks like it is turning off these OMP macros when OMP is present and attempting to include omp.h when HAVE_OMP is false?

Reply:
This was a bug, fixed.